data/json/recipes/armor/head.json

```diff
@@ -1381,5 +1381,18 @@
     "book_learn": [ [ "textbook_armwest", 5 ] ],
     "using": [ [ "forging_standard", 3 ], [ "bronzesmithing_tools", 1 ] ],
     "components": [ [ [ "scrap_bronze", 7 ] ], [ [ "fur", 4 ], [ "leather", 4 ] ] ]
+  },
+  {
+    "result": "attached_ear_plugs_off",
+    "type": "recipe",
+    "category": "CC_ARMOR",
+    "subcategory": "CSC_ARMOR_HEAD",
+    "skill_used": "fabrication",
+    "time": "10 s",
+    "autolearn": true,
+    "components": [
+      [ [ "ear_plugs", 1 ] ],
+      [ [ "string_6", 2 ] ]
```

Comment:
Might want to use filament requirement instead of string here, to enable using of improvised string materials like withered plants or straw.